packages/plugin-ext/src/common/plugin-api-rpc-model.ts

```diff
@@ -512,8 +512,8 @@ export interface RenameLocation {
 }
 
 export interface RenameProvider {
-    provideRenameEdits(model: monaco.editor.ITextModel, position: Position, newName: string): PromiseLike<WorkspaceEdit & Rejection>;
-    resolveRenameLocation?(model: monaco.editor.ITextModel, position: Position): PromiseLike<RenameLocation & Rejection>;
+    provideRenameEdits(model: monaco.editor.ITextModel, position: theia.Position, newName: string): PromiseLike<WorkspaceEdit & Rejection>;
+    resolveRenameLocation?(model: monaco.editor.ITextModel, position: theia.Position): PromiseLike<RenameLocation & Rejection>;
```

Comment:
@eclipse-theia/plugin-system In a different PR there was a problem with this `Position` type too.

TypeScript's old DOM typings used to define some `Position` interface available globally, and I believe that this is what was wrongfully referenced here.

I replaced it with `theia.Position` because new TypeScript DOM typings don't define it anymore, but I don't understand if that's what's expected here.

Reply:
Actually there's no references to this interface, I'll just remove it.